### PR TITLE
Add -version documentation

### PR DESCRIPTION
**Pull request by erikng**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -125,6 +125,14 @@ String. Path to the intended install location of the payload on the target disk.
 **name**  
 String containing the package name. If this is missing, one is constructed using the name of the package project directory.
 
+If you would like munki-pkg to automatically append a version to each package during build, add `-${version}` to the end of the name.
+
```
Comment (gregneagle) [inline README.md]: ??? why would anyone want the version added _after_ the pkg extension?

The intended behavior is variable substitution...

**Comment by erikng** [comment]
Fixed. My fiance is currently trolling me with Christmas music in the form of cat meows and my concentration was not fully there.

File: README.md [hunk 2]
```diff
@@ -125,6 +125,14 @@ String. Path to the intended install location of the payload on the target disk.
 **name**  
 String containing the package name. If this is missing, one is constructed using the name of the package project directory.
 
+If you would like munki-pkg to automatically append a version to each package during build, add `-${version}` to the end of the name.
```
Comment (gregneagle) [inline README.md]: The tool name is `munkipkg` even if the repo name is "munki-pkg"

Comment (gregneagle) [inline README.md]: I think it's better to describe it as variable substitution:

```You can have munkipkg automatically insert the version into the output package filename by including "${version}" in the value of the name key.```

**Comment by gregneagle** [comment]
And another comment got eaten by GitHub.

**Comment by gregneagle** [comment]
Easier for me to just accept the merge and change it to my liking

**Comment by erikng** [comment]
Yes, please. I feel like I'm being double trolled at the moment.

**Comment by natewalck** [comment]
${version} can live anywhere and the - isn't required. nit nit nit.

**Comment by gregneagle** [comment]
Oh I saw that as well. Thus "Easier for me to just accept the merge and change it to my liking"

**Pull request merged**